### PR TITLE
Undo study

### PR DESCRIPTION
**Pull request by axross**
Related to: https://github.com/agnosticful/saraka-mobile/pull/48

### CODE REVIEW

**Comment by axross** [comment]
@ryotogashi Could you check these changes?

**Comment by axross** [comment]
@ryotogashi Maybe you've overlooked this pull request

File: functions/src/entities/__tests__/Study.test.ts [hunk 1]
```diff
@@ -1,10 +1,41 @@
 import { addMilliseconds } from "date-fns";
 import Certainty from "../Certainty";
-import { createNextStudy, FIRST_INTERVAL } from "../Study";
+import { createNextStudy, FIRST_INTERVAL, sanitizeAsStudy } from "../Study";
+import { Timestamp } from "@google-cloud/firestore";
+
+describe("sanitizeAsStudy()", () => {
+  test("returns a Study", () => {
+    const certainty = Certainty.good;
+    const studiedAt = new Date(2019, 11, 31, 23, 59, 59, 999);
```
Comment (ryotogashi) [inline functions/src/entities/__tests__/Study.test.ts]: @axross  Can you teach me what it means?

Comment (axross) [reply functions/src/entities/__tests__/Study.test.ts]: That means creating Date-type value (2019-12-31 23:59:59 (999 milli sec))

**Comment by axross** [comment]
Thanks!

**Pull request merged**